From 12eec28475af5e65ceb63734ac57f6cf453e2c79 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Tue, 28 Feb 2017 16:12:44 +0100
Subject: [PATCH] bridge: man: fix BPUD typo

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1417289
Upstream Status: iproute2.git commit 400b5404af35f

commit 400b5404af35f58f501780663693886d37c2061e
Author: Vivien Didelot <vivien.didelot@savoirfairelinux.com>
Date:   Wed Jun 29 15:26:10 2016 -0400

    bridge: man: fix BPUD typo

    s/BPUD/BPDU/ in guard description.

    Signed-off-by: Vivien Didelot <vivien.didelot@savoirfairelinux.com>
---
 man/man8/bridge.8 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/man8/bridge.8 b/man/man8/bridge.8
index d654e1c..bf9c2c1 100644
--- a/man/man8/bridge.8
+++ b/man/man8/bridge.8
@@ -256,7 +256,7 @@ STP BPDUs.
 
 .TP
 .BR "guard on " or " guard off "
-Controls whether STP BPUDs will be processed by the bridge port. By default,
+Controls whether STP BPDUs will be processed by the bridge port. By default,
 the flag is turned off allowed BPDU processing. Turning this flag on will
 cause the port to stop processing STP BPDUs.
 
-- 
1.8.3.1

